drivers/pinctrl/pinctrl_ti_k3.c

```diff
@@ -6,31 +6,43 @@
 
 #define DT_DRV_COMPAT ti_k3_pinctrl
 
+#include <k3_mmr.h>
+#include <zephyr/device.h>
 #include <zephyr/arch/cpu.h>
 #include <zephyr/drivers/pinctrl.h>
 
 #define PINCTRL_NODE DT_NODELABEL(pinctrl)
+#define PADCFG_NODE  DT_NODELABEL(padcfg)
 
 static struct pinctrl_ti_k3_dev_data {
 	DEVICE_MMIO_RAM;
 } pinctrl_ti_k3_dev;
 
 static struct pinctrl_ti_k3_cfg_data {
 	DEVICE_MMIO_ROM;
-} pinctrl_ti_k3_cfg = {
-	DEVICE_MMIO_ROM_INIT(PINCTRL_NODE)
-};
+} pinctrl_ti_k3_cfg = {DEVICE_MMIO_ROM_INIT(PINCTRL_NODE)};
```

Comment:
Unrelated code format change? Should remove these.

Reply:
Ah, ran clang-format on changed files, thought it would be good to leave these changes, but will remove them from now on 👍